SportShoesStore.cpp

```diff
@@ -0,0 +1,64 @@
+#include <iostream>
+#include "SportShoesStore.h"
+#include "Sneakers.h"
+
+
+void SportShoesStore::sortprice() {
+	int len = s.size();
+	cout << "Sort by price:" << endl;
+	for (int i = 0; i < len - 1; i++)
+	{
+		for (int j = 0; j < len - i - 1; j++)
+		{
+			if (s[j].getPrice() > s[j + 1].getPrice())
+			{
+				std::swap(s[j], s[j + 1]);
+			}
+		}
+	}
+	for (int i = 0; i < len; i++)
+	{
+		cout << "Brand:" << s[i].getBrand() << "| Price: " << s[i].getPrice() << endl;
+	}
+	cout << "----------------------------------" << endl;
+}
+
+void SportShoesStore::sortquantity() {
```

Comment:
Please edit method name to more readable (sortQuantity or SortQuantity or sort_quantity)